js/bitget.js

```diff
@@ -2350,23 +2352,13 @@ module.exports = class bitget extends Exchange {
         return this.parseOrders (data, market, since, limit);
     }
 
-    async fetchClosedOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
-        /**
-         * @method
-         * @name bitget#fetchClosedOrders
-         * @description fetches information on multiple closed orders made by the user
-         * @param {string} symbol unified market symbol of the market orders were made in
-         * @param {int|undefined} since the earliest time in ms to fetch orders for
-         * @param {int|undefined} limit the maximum number of  orde structures to retrieve
-         * @param {object} params extra parameters specific to the bitget api endpoint
-         * @returns {[object]} a list of [order structures]{@link https://docs.ccxt.com/en/latest/manual.html#order-structure}
-         */
+    async fetchOrdersByStatus (status, symbol = undefined, since = undefined, limit = undefined, params = {}) {
```

Comment:
@Dan-krm I don't think these edits are correct, `fetchOrderByStatus` is meant to be used when the exchange provides an endpoint that returns all kinds of orders (open, closed, canceled, ...) and allows for filtering using some parameter in the request or URL (check for instance kucoin's implementation).  Here we're using an endpoint that returns only "closed/canceled" orders

Reply:
@Dan-krm, I'd be ok if you renamed it to something else other than `fetchOrderByStatus`so that the users don't think this is the unified `fetchOrderByStatus` due to the reasons mentioned earlier but we can also ask @kroitor's opinion too

Reply:
> @Dan-krm, I'd be ok if you renamed it to something else other than `fetchOrderByStatus`so that the users don't think this is the unified `fetchOrderByStatus` due to the reasons mentioned earlier but we can also ask @kroitor's opinion too

@carlosmiei I think of `fetchOrdersByStatus` as a helper method and not a unified method because it's not listed under has for any of the exchanges and it's not in the manual, we're not really expecting users to use `fetchOrdersByStatus` directly so I think we can be pretty flexible with our use of it. I can rename it though if we decide that's necessary